src/app/plugins/platform/JavaScript/index.ts

```diff
@@ -107,6 +108,8 @@ class JavaScript extends Generator {
     // Sass compilation and plugins
     this.npmInstall(
       [
+        'sass',
+        'fibers',
```

Comment:
Looks like sass-loader wants them:

```
npm WARN sass-loader@8.0.0 requires a peer of sass@^1.3.0 but none is installed. You must install peer dependencies yourself.
npm WARN sass-loader@8.0.0 requires a peer of fibers@>= 3.1.0 but none is installed. You must install peer dependencies yourself.
```

Reply:
Based on the docs, it seems like those two packages are only needed if you want to use Dart Sass. I'd suggest we avoid that since Gesso currently uses node-sass. We can revisit should Gesso's Sass implementation change. We can document that there will be two `sass-loader`-related warnings that are safe to ignore.

Reply:
All righty then. Where's the best place to document something like that?

Reply:
Let's put it in our Confluence for now. There are plans to organize the docs to bring them out, but we're not there yet.

Reply:
👍 Noted on https://forumone.atlassian.net/wiki/spaces/TECH/pages/738918488/Generating+and+Running+JavaScript+Projects